src/components/ResetFormButton/ResetFormButton.tsx

```diff
@@ -0,0 +1,30 @@
+import Button from "@/components/Button/Button"
```

Comment:
Consider updating base custom button with additional props instead of creating a new component just for reseting the form